### PR TITLE
Add @wesleyc94

### PR DESCRIPTION
**Pull request by wesleyc94**

### CODE REVIEW

File: site/class-2-style/wesleyc94/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+function render(b, c) {
+    if (b) throw b;
+    var a = svg.selectAll('.arc').data(pie(c)).enter().append('g').attr('class', 'arc');
+    a.append('path').attr('d', arc).style('fill', fill), a.append('text').attr('transform', transform).attr('dy', '.35em').text(age);
```
Comment (wooorm) [inline site/class-2-style/wesleyc94/index.js]: This isn’t very pretty :(

**Comment by wooorm** [comment]
you also included the readme assignment on this branch, right?

**Comment by wesleyc94** [comment]
Yes, but i don't know how I did it haha. I hope its ok :)

**Comment by wooorm** [comment]
@wesleyc94 WEIRD!

Maybe delete your fork and fork again? And then don’t commit on master again, and don‘t merge your own branches!

**Pull request merged**